lib/engines/hcktest/hcktest.rb

```diff
@@ -226,29 +226,44 @@ def tag
       end
     end
 
-    def manual_run
+    def dump_only_run
       ResourceScope.open do |scope|
         run_studio(scope, { dump_only: true })
         run_clients(scope, { dump_only: true })
       end
     end
 
+    def wait_manual
+      return unless @project.options.test.manual
+
+      @project.logger.info('AutoHCK started in manual mode. Waiting for manual exit.')
+      loop do
+        @project.logger.info("Type 'exit' and press ENTER to exit AutoHCK")
+
+        break if $stdin.gets&.chomp == 'exit'
+      end
+      @project.logger.info('Manual exit. AutoHCK will continue.')
+    end
+
     def auto_run
       ResourceScope.open do |scope|
         run_and_configure_setup scope
         client = @client1
         client.run_tests
+
+        wait_manual
```

Comment:
It is awkward to call `wait_manual` from `auto_run`.

Reply:
renamed